.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+fail_fast: false
+repos:
+  - repo: https://github.com/pocc/pre-commit-hooks
+    rev: master
+    hooks:
+      - id: clang-format
+        args:
+          - -i
+  - repo: https://github.com/prettier/prettier
+    rev: master
+    hooks:
+      - id: prettier
```

Comment:
Presumably `prettier` runs on any file it supports - not just docs?

Reply:
yup: https://github.com/pre-commit/mirrors-prettier

> By default, all files are passed to prettier, if you want to limit the file list, adjust types / files:
```
    -   id: prettier
        types: [javascript]
```